package.json

```diff
@@ -36,11 +36,11 @@
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-register": "^6.18.0",
-    "chai": "^3.5.0",
-    "coveralls": "^2.11.14",
-    "eslint": "^4.4.1",
-    "eslint-plugin-prettier": "^2.1.2",
-    "mocha": "^3.1.2",
+    "chai": "^4.1.1",
```

Comment:
Set testing was broken in old unit testing libs. I upgraded these together to the latest version to be safe.